frontend/front/README.md

```diff
@@ -127,3 +127,44 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `yarn build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+
+######################################################################################################################
+
+From Randy DAbbraccio 6.22.2023
+
+To build a Docker container in a LINUX system and run it:
+
+Git clone repo
+
+Goto /frontent/front and run yarn install
+
+Run 'npm run build' to make a production version
+
+Goto new /build folder
+
+Install Docker code if not already
+
+Make a Dockerfile and put the following in it:
```

Comment:
Could you put this in a Dockerfile and commit that? Then we could reference it here instead of having folks make their own.

Reply:
@mattdelsordo I put the file in there plus a 79mb container. We need decide if we want to keep that in main....

Reply:
The container should not be checked in to git, we should leave it up to the dev or the CI to build a container from the Dockerfile when needed.

Reply:
@mattdelsordo 

New commit.

I removed the TAR file and updated a script. It should install in Linux. Please read the README.md in docker-builds to see if it works for you. Thanks